web/src/pages.rs

```diff
@@ -75,38 +79,54 @@ pub async fn article_detail(
     .await?;
 
     let markdown = if let Some(page_snapshot) = page_snapshot {
-        sqlx::query_as!(
+        let markdown = sqlx::query_as!(
             Markdown,
             "SELECT * FROM markdown WHERE page_snapshot_id = $1",
             page_snapshot.page_snapshot_id
         )
         .fetch_optional(&state.db)
-        .await?
+        .await?;
+
+        if let Some(markdown) = markdown {
+            let categories = sqlx::query_as!(
+                Category,
+                "SELECT * FROM category WHERE markdown_id = $1",
+                markdown.markdown_id
+            )
+            .fetch_all(&state.db)
+            .await?;
+            Some((markdown, categories))
+        } else {
+            None
+        }
     } else {
         None
     };
-    info!("Fetched Article: {:?}", article);
 
-    info!("Fetched markdown MD: {:?}", markdown);
-
-    Ok(t.render(maud::html! {
-        @if let Some(markdown) = markdown {
-            p { (markdown.summary) }
-        } @else {
+    let rendered_html = if let Some((markdown, categories)) = markdown {
+        maud::html! {
+            ul {
+                @for category in categories {
+                    li { b { "Category: " } (category.category.unwrap_or("No category".to_string())) }
```

Comment:
`.unwrap_or() `method is used in this snippet to handle the case where category.category might be` None.`